pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbbackup/fullalloydbbackup/create.yaml

```diff
@@ -18,11 +18,11 @@ metadata:
   name: alloydbbackup-${uniqueId}
 spec:
   clusterNameRef:
-    external: "projects/${projectId}/locations/us-east1/clusters/alloydbcluster${uniqueId}"
-  location: us-east1
+    external: alloydbcluster-${uniqueId}
```

Comment:
The value of `external` field should be the value accepted by `clusterName` field in the GCP API, which should be the relative resource name instead of the resource ID: https://cloud.google.com/alloydb/docs/reference/rest/v1beta/projects.locations.backups

I assume you want to use `name` instead of `external`?

Reply:
Yes, thanks. Updated field to use name instead of external